pkg/seeds/seeds.go

```diff
@@ -91,12 +91,12 @@ func SeedRepository(db *gorm.DB, size int) error {
 	// Add size randome Repository entries
 	countRecords := 0
 	for i := 0; i < size; i++ {
-		lastReadTime, lastReadError := randomLastRead()
+		lastIntrospectionTime, lastIntrospectionError := randomLastRead()
```

Comment:
depending on the above comment, it could need more variables for the returning values.